parser/parser.go

```diff
@@ -2138,9 +2146,9 @@ func (p *parser) parseLambdaExpr(allowCmd, allowRangeExpr bool) ast.Expr {
 			RhsHasParen: rhsHasParen,
 		}
 	}
-	if _, ok := x.(*tupleExpr); ok {
-		panic("TODO: tupleExpr")
-	}
+	// if _, ok := x.(*tupleExpr); ok {
+	// 	panic("TODO: tupleExpr")
+	// }
```

Comment:
where to check this unexpected expression?

Reply:
parseLambdaExpr add allowTuple param